### PR TITLE
Change upgrade playbooks to use 4.0

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

File: playbooks/common/openshift-cluster/upgrades/v4_0/upgrade_control_plane_part2.yml [hunk 1]
```diff
@@ -116,16 +116,3 @@
   - import_role:
       name: openshift_web_console
       tasks_from: remove_old_asset_config.yml
-
-# This is a one time migration. No need to save it in the 3.11.
```
Comment (michaelgugino) [inline playbooks/common/openshift-cluster/upgrades/v4_0/upgrade_control_plane_part2.yml]: Should this be removed in 3.11?

File: playbooks/byo/openshift-cluster/upgrades/v4_0/upgrade_nodes.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (michaelgugino) [inline playbooks/byo/openshift-cluster/upgrades/v4_0/upgrade_nodes.yml]: I don't know why we still have the byo plays, why add this here now?

File: playbooks/common/openshift-cluster/upgrades/v4_0/upgrade_control_plane_part2.yml (hunk 1 above)
Comment (vrutkovs) [reply playbooks/common/openshift-cluster/upgrades/v4_0/upgrade_control_plane_part2.yml]: Yeah, I'll include this in https://github.com/openshift/openshift-ansible/pull/10031 and test it there

Comment (sdodson) [reply playbooks/common/openshift-cluster/upgrades/v4_0/upgrade_control_plane_part2.yml]: Given that a lot of this should go away in 4.0 I'm not terribly worried about doing a lot of immediate cleanup. Just that we can get 3.11 to 4.0 tests into CI until such a time that we begin active 4.0 development in openshift-ansible.

File: playbooks/byo/openshift-cluster/upgrades/v4_0/upgrade_nodes.yml (hunk 2 above)
Comment (vrutkovs) [reply playbooks/byo/openshift-cluster/upgrades/v4_0/upgrade_nodes.yml]: IIUC tox tests want these

Lets see what happens if I remove those

Comment (vrutkovs) [reply playbooks/byo/openshift-cluster/upgrades/v4_0/upgrade_nodes.yml]: oh, I forgot to rename this one. Fixed

**Comment by vrutkovs** [comment]
Jenkins got stuck, previous build passed - https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-ansible/10080/test_pull_request_openshift_ansible_extended_conformance_install/11532/

**Pull request merged**